include/arch/riscv/common/linker.ld

```diff
@@ -50,18 +68,6 @@
 MEMORY
 {
 #ifdef CONFIG_XIP
-#if DT_NODE_HAS_COMPAT_STATUS(DT_CHOSEN(zephyr_flash), soc_nv_flash, okay)
-#define ROM_BASE DT_REG_ADDR(DT_CHOSEN(zephyr_flash))
-#define ROM_SIZE DT_REG_SIZE(DT_CHOSEN(zephyr_flash))
-#elif DT_NODE_HAS_COMPAT_STATUS(DT_CHOSEN(zephyr_flash), jedec_spi_nor, okay)
-/* For jedec,spi-nor we expect the spi controller to memory map the flash
- * and for that mapping to be the second register property of the spi
- * controller.
- */
-#define SPI_CTRL DT_PARENT(DT_CHOSEN(zephyr_flash))
-#define ROM_BASE DT_REG_ADDR_BY_IDX(SPI_CTRL, 1)
-#define ROM_SIZE DT_REG_SIZE_BY_IDX(SPI_CTRL, 1)
-#endif
     ROM (rx)  : ORIGIN = ROM_BASE, LENGTH = ROM_SIZE
 #endif
     RAM (rwx) : ORIGIN = CONFIG_SRAM_BASE_ADDRESS, LENGTH = KB(CONFIG_SRAM_SIZE)
```

Comment:
Can we also rewrite:
RAM (rwx) : ORIGIN = ROM_BASE, LENGTH = ROM_SIZE

Reply:
Do you mean that rewrite to `RAM (rwx) : ORIGIN = RAM_BASE, LENGTH = RAM_SIZE` for using same style of ROM/RAM's base/size.
If so, it's ok. I can update it later.

Reply:
Yes, like as other arch...
But it's out of this PR topic, please forget. Sorry for confusing.

Reply:
No worry, I just want to make sure that it's typo.
Fixed it.